### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by ReddicusDragon**
# Document the changes in your pull request
Noticed SuperSlayer0's code had many typos, went around and fixed some grammar and other issues.

# Changelog
:cl:  
spellcheck: fixed a few typos and grammar issues
/:cl:

### CODE REVIEW

File: code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm [hunk 1]
```diff
@@ -222,7 +222,7 @@
 //Call brass bow: Creates or calls a brass bow, which is capable of firing redlight arrows.
 /datum/action/innate/call_weapon/brass_bow
 	name = "Call Brass Bow"
-	desc = "Calls a brass bowcapable of firing redlight arrows, doing moderate damage but recharges after use. The bow will recharge after 1.5 seconds.."
+	desc = "Calls a brass bow capable of firing redlight arrows that do moderate damage and recharge after use. The bow will recharge after 1.5 seconds."
```
Comment (ynot01) [inline code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm]: The arrows do not recharge after use, the bow does

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -1493,7 +1493,7 @@
 	inherent_traits = list(TRAIT_NOHARDCRIT,TRAIT_NOSOFTCRIT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_RADIMMUNE,TRAIT_GENELESS,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER,TRAIT_NOHUNGER,TRAIT_NOGUNS)
 	changesource_flags = MIRROR_BADMIN 
 	random_eligible = FALSE // Hell no
-	info_text = "As a <span class='danger'>Supermatter Golem</span>, you dust almost any physical objects that interact with you. However, you take half more brute damage, three more burn damage and explode on death."
+	info_text = "As a <span class='danger'>Supermatter Golem</span>, you dust almost any physical object that interact with you, while taking half times more brute damage and three times more burn damage. You explode on death."
```
Comment (ynot01) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]:
```suggestion
	info_text = "As a <span class='danger'>Supermatter Golem</span>, you dust almost any physical object that interacts with you, while taking half as much brute damage and three times more burn damage. You also explode on death."
```

File: code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm (hunk 1 above)
Comment (ReddicusDragon) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm]: Semantics.

Comment (ReddicusDragon) [reply code/modules/antagonists/clockcult/clock_scriptures/scripture_scripts.dm]: Will resolve when I get home

**Review by MajManatee — APPROVED** [review]
Semantics anyways

**Pull request merged**